src/pudl/transform/eia860.py

```diff
@@ -748,3 +748,51 @@ def clean_boilers_eia860(
     )
 
     return b_df
+
+
+@asset
+def clean_emissions_control_equipment_eia860(
+    raw_emissions_control_equipment_eia860: pd.DataFrame,
+) -> pd.DataFrame:
+    """Pull and transform the emissions control equipment table."""
+    # Replace empty strings, whitespace, and '.' fields with real NA values
+    emce_df = pudl.helpers.fix_eia_na(raw_emissions_control_equipment_eia860)
+
+    # Fix bad months
+    emce_df["operating_month"] = emce_df["operating_month"].replace({"88": "8"})
+    emce_df["retirement_month"] = emce_df["retirement_month"].replace({"True": np.nan})
+
```

Comment:
I'm not sure what to do about the date columns pertaining to operation and retirement. There's a month and a year column for each of them. In the past we've mostly converted these types of columns into a single Datetime column using the helper `convert_to_date` function. The issue with this is that A) it adds a "day" to the date that just ends up being the first of the month -- not a huge deal and relatively standard practice, just worth nothing. More importantly, B) it will null out rows that have a `year` value but not a `month` value. Even if we decide to change that, Datetime still requires M/D/Y and I feel less good about defaulting to January as I do the first day of the month. My question here is whether we want to keep these columns separate (month/year) or create a new Datetime column and if so, what we should do about rows with no month data.

Reply:
a) we default to first day of the month all over the place so i am pro keeping that convention and keeping it standard.
b) yeesh this is worse for sure.... I think these should either be null of January. We want a date column as this is consistent throughout our data. I'm personally pro January of year so there is some info in there but idk

Reply:
Ok that's what I'll do. I'll make sure the metadata for this field explains this caveat.

Reply:
I took a closer look at the instances where there was an `operating_year` but not `operating_month` and was able to spot fix all of them based on future years. So I feel cozy doing the date thing!